app/components/delivery_partners/participants/table_row.html.erb

```diff
@@ -0,0 +1,32 @@
+<tr class="govuk-table__row">
+  <td class="govuk-table__cell">
+    <%= full_name %>
+  </td>
+  <td class="govuk-table__cell">
+    <%= email %>
+  </td>
+  <td class="govuk-table__cell">
+    <%= teacher_profile.trn %>
+  </td>
+  <td class="govuk-table__cell">
+    <%= user_description %>
+  </td>
+  <td class="govuk-table__cell">
+    <%= school_cohort&.school&.active_partnerships[0]&.lead_provider&.name %>
```

Comment:
i think this should be closer along the lines of:
```
participant_profile.induction_records.active.latest.induction_programme.partnership...
```
due to the change of modelling